audio_context.html

```diff
@@ -0,0 +1,321 @@
+  </header>
+
+  <fieldset>
+
+    <legend>Sweep</legend>
+
+    <label>
+      Attack
+      <input type="range" name="attack" min="0" max="1" value="0.2" step="0.1">
+    </label>
+
+    <label>
+      Release
+      <input type="range" name="release" min="0" max="1" value="0.5" step="0.1">
+    </label>
+
+    <fieldset>
+
+      <legend hidden>Checkboxes</legend>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+      <label>
+        <input name=pads type="checkbox">
+      </label>
+
+    </fieldset>
+
+  </fieldset>
+
+  <fieldset>
+
+    <legend>Pulse</legend>
+
+    <label>
+      Hz
+      <input type="range" name="hertz" min="660" max="1320" value="880" step="1">
+    </label>
+
+    <label>
+      LFO
+      <input type="range" name="lfo" min="20" max="40" value="30" step="1">
+    </label>
+
+  <fieldset>
+
+    <legend hidden>Checkboxes</legend>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+  </fieldset>
+
+  </fieldset>
+
+  <fieldset>
+    <legend>Noise</legend>
+
+    <label>
+      Duration
+    <input type="range" name="duration" min="0" max="2" value="1" step="0.1">
+    </label>
+
+    <label>
+      Band
+    <input type="range" name="band" min="400" max="1200" value="1000" step="5">
+    </label>
+
+    <fieldset>
+
+    <legend hidden>Checkboxes</legend>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+  </fieldset>
+
+  </fieldset>
+
+  <fieldset>
+
+    <legend>DTMF</legend>
+
+      <label>
+        Rate
+      <input type="range" name="rate" min="0.1" max="2" value="1" step="0.1">
+      </label>
+
+  <fieldset>
+
+    <legend hidden>Checkboxes</legend>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+    <label>
+      <input name=pads type="checkbox">
+    </label>
+
+  </fieldset>
+
+  </fieldset>
+
+  </form>
+
+</main>
+  <script>
+    function initialize() {
+      const form = document.querySelector ('main > form')
+        , fieldsets = form.querySelectorAll ('fieldset')
+
+      console.log(fieldsets)
+
+      // https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event
+      form.addEventListener ('submit', submit)
+
+    for (let fieldset of fieldsets)
+    configure(fieldset)
```

Comment:
One more indent.  `for` loops typically have curly braces but if you only have one line of code you can forego curly braces.  Typically indent that one line 1 level deeper than the `for` loop for readability.

Reply:
👍🏽